src/app/calculator/compressed-air/air-flow-conversion/air-flow-conversion.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnInit, ViewChild, ElementRef, HostListener, Input } from '@angular/core';
+import { Settings } from '../../../shared/models/settings';
+import { AirFlowConversionService } from './air-flow-conversion.service';
+import { SettingsDbService } from '../../../indexedDb/settings-db.service';
+import { Subscription } from 'rxjs';
+import { AirFlowConversionOutput } from '../../../shared/models/compressed-air/compressed-air';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-air-flow-conversion',
+  templateUrl: './air-flow-conversion.component.html',
+  styleUrls: ['./air-flow-conversion.component.css']
+})
+export class AirFlowConversionComponent implements OnInit {
+  @Input()
+  settings: Settings;
+  headerHeight: number;
+  currentField: string = 'default';
+  form: FormGroup;
+
+  airFlowConversionOutput: AirFlowConversionOutput;
+  airFlowConversionInputSub: Subscription;
+  airFlowConversionOutputSub: Subscription;
+  currentFieldSub: Subscription;
+
+  @ViewChild('leftPanelHeader', { static: false }) leftPanelHeader: ElementRef;
+
+  @HostListener('window:resize', ['$event'])
+  onResize(event) {
+    this.resizeTabs();
+  }
+
+  
+  constructor(private airFlowConversionService: AirFlowConversionService,
+              private settingsDbService: SettingsDbService) { }
+
+  ngOnInit(): void {
+    if (!this.settings) {
+      this.settings = this.settingsDbService.globalSettings;
+    }
+    this.airFlowConversionService.initDefaultEmptyInputs(this.settings);
+    this.airFlowConversionService.initDefaultEmptyOutputs();
+    this.initSubscriptions();
+  }
+
+  ngOnDestroy() {
+    this.airFlowConversionInputSub.unsubscribe();
+  }
+
+  ngAfterViewInit() {
+    setTimeout(() => {
+      this.resizeTabs();
+    }, 100);
+  }
+
+  resizeTabs() {
+    if (this.leftPanelHeader.nativeElement.clientHeight) {
+      this.headerHeight = this.leftPanelHeader.nativeElement.clientHeight;
+    }
+  }
+
+  initSubscriptions() {
+    this.airFlowConversionInputSub = this.airFlowConversionService.airFlowConversionInput.subscribe(value => {
+      this.calculate();
+    })
+    this.airFlowConversionOutputSub = this.airFlowConversionService.airFlowConversionOutput.subscribe(value => {
```

Comment:
These will probably get moved with the addtional components but make sure you unsubscribe all subscriptions.